test_runner/src/main/kotlin/ftl/cli/firebase/test/ios/IosRunCommand.kt

```diff
@@ -60,3 +60,8 @@ class IosRunCommand : CommonRunCommand(), Runnable {
     )
     var dumpShards: Boolean = false
 }
+
+private fun IosArgs.logArgs() = also {
+    setupLogLevel()
```

Comment:
`setupLogLevel` looks like an independent operation not only related to args printing. IMO it should be called before `IosArgs.load`.

Reply:
Changed! Thanks!